state_machine/applications/flight/lib/settings.py

```diff
@@ -0,0 +1 @@
+TX_ALLOWED = False
```

Comment:
I really like this. Not in this PR, but later It would be a convenient spot to put if we were using the fsk or lora protocols to make switching between them easier. I made something similar to this a few weeks ago on a local branch when I was doing a lot of radio testing for both protocols.

Reply:
Yeah that's a good idea